crates/typst/src/text/mod.rs

```diff
@@ -90,7 +90,8 @@ pub(super) fn define(global: &mut Scope) {
 /// ```
 #[elem(Debug, Construct, PlainText, Repr)]
 pub struct TextElem {
-    /// A font family name or priority list of font family names.
+    /// A font family name or priority list of font family names. Run
+    /// `typst fonts` to see the fonts that Typst has discovered on your system.
```

Comment:
I think this information would fit better with the bullet point starting with `Locally, ...` below.

Reply:
Yes makes a lot of sense! I forgot about the web app. Updated in e7233b6

Reply:
In browser, you could click the `Ag` button to discover fonts on webapp, which is also worth to be mentioned at once here imo. At least I didn't find it for months even if webapp had related tips for me.

Reply:
@Myriad-Dreamin can you suggest an exact text since I'm not familiar with your suggestion (or open a separate PR)

Reply:
I just noticed that it's already in there: "In the web app, you can see the list of available fonts by clicking on the "Ag" button. You can provide additional fonts by uploading `.ttf` or `.otf` files into your project. They will be discovered automatically.", so no further explanations should be necessary.